spec/std/isa/inst/Zfh/fmv.h.x.yaml

```
@@ -12,7 +12,8 @@ description: |
   from the lower 16 bits of integer register `xs1` to the floating-point
   register `fd`. The bits are not modified in the transfer, and in particular,
   the payloads of non-canonical NaNs are preserved.
-definedBy: F
+definedBy:
+  anyOf: [Zfh, Zfhmin, Zhinx]
```

Great catch on Zfhmin and Zhinx in addition to what the provoking issue claimed, Zfh.

Indeed, since Zfhmin is a proper subset of Zfh, you can simplify this to:
```suggestion
  anyOf: [Zfhmin, Zhinx]
```

---

Sure sir updating it